library/swift/src/EnvoyClient.swift

```diff
@@ -69,7 +69,8 @@ public final class EnvoyClient: NSObject {
 
 extension EnvoyClient: HTTPClient {
   public func send(_ request: Request, handler: ResponseHandler) -> StreamEmitter {
-    let httpStream = self.engine.startStream(with: handler.underlyingCallbacks)
+    let httpStream = self.engine.startStream(
+      with: handler.underlyingCallbacks, bufferForRetry: request.retryPolicy != nil)
```

Comment:
nice, the end-user does not have to know about the tie between the stream option and retrying.

Reply:
Yep!